README.md

```diff
@@ -52,7 +52,14 @@ by one line of code.
 
 ## Installation
 
-You can install the development version from [GitHub](https://github.com/) with:
+You can install the released version of rimager from
+[CRAN](https://CRAN.R-project.org) with:
```

Comment:
Since we won't be releasing on cran should we remove this?

Reply:
Yes